Meme/Meme/Models/Template.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Template.swift
+//  Meme
+//
+//  Created by Mesfin Bekele Mekonnen on 7/29/19.
+//  Copyright © 2019 Mesfin Bekele Mekonnen. All rights reserved.
+//
+
+import Foundation
+
+struct Template {
+    let name: String
```

Comment:
you can probably now reduce Template, TemplateExample, Meme down to one object since you no longer need multiple trips to the API.

Reply:
I deleted TemplateExample.